logprep/processor/pseudonymizer/processor.py

```diff
@@ -346,10 +346,12 @@ def _pseudonymize_value(self, value: str, pseudonyms: List[dict]) -> str:
     def _replace_regex_keywords_by_regex_expression(self):
         for rule in self._specific_rules:
             for dotted_field, regex_keyword in rule.pseudonyms.items():
-                rule.pseudonyms[dotted_field] = self._regex_mapping[regex_keyword]
+                if regex_keyword in self._regex_mapping:
```

Comment:
For the sake of readability consider to explicitly check the keys of `self._regex_mapping` like this

```python
                if regex_keyword in self._regex_mapping.keys():

```

Otherwise if this is a suggested and faster way of doing this then ignore my suggestion.

Reply:
the second suggestion will result in a pylint warning, because the first suggestion is the way to go ;-)

Reply:
okay, sometimes I do not agree with pylint but that's the way the cookie crumbles